drupal.env.sample

```diff
@@ -25,3 +25,7 @@ XDEBUG_IP=127.0.0.1
 
 # Prevent memory limit issues when installing new packages the composer
 COMPOSER_MEMORY_LIMIT=-1
+
+# Setup variables for Drupal phpunit testing.
+SIMPLETEST_BASE_URL=http://drupal:8080
```

Comment:
The changes introduced in PR-754 change this to `prisoner-content-hub-backend`

Reply:
Should we rename the files as well in that case?  `drupal.env.sample` -> `prisoner-content-hub-backend.env.sample`

Reply:
we probably should, I missed renaming those 🤦

Reply:
I'll update them in this PR